crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -490,6 +482,20 @@ where
         ));
     }
 
+    fn update_archetypes(&mut self, world: &World) {
```

Comment:
is there a perf improment if you #[inline] this function?

Reply:
The benchmark is too noisy to really answer this well, but I suspect the answer is no in practise for normal use cases - this is now mostly only going to be called from other code in the crate and so it will get inlined anyway if LLVM thinks it's worth it.